### ISSUE TITLE
Test the Blazor UI with the Release Mode / Publish

### ISSUE THREAD

**Issue by hikalkan** [abpframework/abp#5646]
see https://github.com/abpframework/abp/issues/394#issuecomment-699166579

**Comment by hikalkan** [comment]
I deployed from dev branch (with .net 5) to my local IIS and has no problem.

**Comment by hikalkan** [comment]
I have the same problem with https://github.com/abpframework/abp/issues/394#issuecomment-699166579 for version `3.3.0-rc.1` when I run dotnet publish:

````bash
dotnet publish -o C:\Temp\Publishes\Blazor -c Release
````

**Comment by hikalkan** [comment]
This seems problem related to the runtime.

See https://github.com/dotnet/aspnetcore/issues/18701 that is moved to https://github.com/mono/mono/issues/18651

The current solution is https://github.com/mono/mono/issues/18651#issuecomment-646276014

Add the following line into the `PropertyGroup` section of your Blazor project.

````xml
<BlazorWebAssemblyEnableLinking>false</BlazorWebAssemblyEnableLinking>
````

Applied for the startup template: https://github.com/abpframework/abp/commit/9e8817f9e323282bc1e307f6a294113b154a7f1a

(also fixed for version 3.3.0 that will be released in the next week)
